### DISCUSSION TITLE
Feature Request: Roles for authors/contributors

### DISCUSSION THREAD

**Discussion by iamjmulligan** [bookwyrm-social/bookwyrm#2534] [Ideas]
I do wish rather than just placing all people involved with a work as narrators that there would be different roles like there is on Librarything. I'd find this especially useful to delineate narrators on audiobooks, and pencillers and colourists with graphic novels.

**Comment by mouse-reeve** [comment]
That's in progress! You can follow #1120 for updates

**Reply by iamjmulligan** [reply]
Of course. Should have expected that.

There's a bunch of basic features that I'd want to see that are all pretty obvious like that. Things like:

Being able to search by Author, Subject and Series
Being able to click on the Series to display all the entries in that series.
Being able to click on a Subject to display all the works with that Subject, and being able to filter that to show only books in that persons library

Alot of my desire is various ways to look into my own library and sort it. Sorting by what was completed last or most recently published being a couple of those.

Another thing is merging author pages, as I've frequently noticed when adding books as one author has multiple author pages that I just wish I could merge.
